build.py

```diff
@@ -40,6 +41,14 @@ def __init__(self):
         VISUAL STUDIO: %s
         """ % (self.gcc_compiler.versions, self.clang_compiler.versions, self.visual_compiler.versions))
 
+    def _get_conan_target_version(self):
+        conan_version_file = "conan_version.txt"
+        if os.path.isfile(conan_version_file):
+            for line in reversed(list(open(conan_version_file))):
+                match = re.search("Bumped for Conan (.*)", line.rstrip())
```

Comment:
Wouldn't a variable in some Python file be better than this text file?

Reply:
Yes, totally!

Reply:
Done, thank you!!